opm/common/OpmLog/EclipsePRTLog.hpp

```diff
@@ -29,16 +29,21 @@ namespace Opm {
 class EclipsePRTLog : public StreamLog {
 
 public:
-    using StreamLog::StreamLog;
 
     void addTaggedMessage(int64_t messageType, const std::string& messageTag, const std::string& message);
 
     size_t numMessages(int64_t messageType) const;
 
     ~EclipsePRTLog();
 
+    EclipsePRTLog(const std::string& logFile , int64_t messageMask,
```

Comment:
> When you use a parent's constructor in the init list it falls somewhere in between, but syntactically and largely conceptually it's delegating.

Are you talking about the case of
```C++
Derived::Derived(const X& x, const Y& y)
    : Base(x)
{
    init(x, y);
}
```
:question: 

If so, that's nothing more than initialising the base portion of the derived object.  Even C++98, long before delegating constructors, requires that this be done.